packages/@ionic/cli/src/commands/package/deploy.ts

```diff
@@ -0,0 +1,251 @@
+import {
+  CommandLineInputs,
+  CommandLineOptions,
+  LOGGER_LEVELS,
+  combine,
+  validators,
+} from '@ionic/cli-framework';
+import { columnar } from '@ionic/cli-framework/utils/format';
+import { sleep } from '@ionic/utils-process';
+import * as chalk from 'chalk';
+
+import { CommandMetadata } from '../../definitions';
+import { isSuperAgentError } from '../../guards';
+import { input, strong } from '../../lib/color';
+import { Command } from '../../lib/command';
+import { FatalException } from '../../lib/errors';
+
+import { PackageBuild } from './build';
+
+interface BinaryDeployment {
+  id: number;
+  user: any;
+  build: any;
+  type: string;
+  distributionCredential: any;
+  destination: string;
+  message: string;
+  distributionBuildId: number;
+  status: string;
+}
+
+interface DistributionBuild {
+  job_id: number;
+  id: string;
+  caller_id: number;
+  created: string;
+  state: string;
+  distribution_credential_name: string;
+  package_build: PackageBuild;
+  binary_deployment: BinaryDeployment;
+  distribution_trace: string;
+}
+
+export class BuildCommand extends Command {
```

Comment:
Small thing, but maybe rename to `DeployCommand`?

Reply:
sorry, copy/paste leftover

Reply:
pushed change